telegram/ext/callbackqueryhandler.py

```diff
@@ -81,7 +81,8 @@ class CallbackQueryHandler(Handler[Update]):
             DEPRECATED: Please switch to context based callbacks.
         pattern (:obj:`str` | `Pattern`, optional): Regex pattern. If not :obj:`None`, ``re.match``
             is used on :attr:`telegram.CallbackQuery.data` to determine if an update should be
-            handled by this handler.
+            handled by this handler. If :attr:`telegram.CallbackQuery.data` is not present, the
+            CallbackQuery update will not be handled.
```

Comment:
```suggestion
            :class:`telegram.CallbackQuery` update will not be handled.
```
or just "the update" …

Reply:
nah I like the link